src/super_gradients/training/utils/collate_fn/detection.py

```diff
@@ -0,0 +1,35 @@
+from typing import Tuple, Dict
+
+import torch
+from torch.utils.data._utils.collate import default_collate
+
+
+class DetectionCollateFN:
```

Comment:
If you already decided to concentrate the different collate fns here- make sure to add the missing ones (discussed in earlier comment).